Terran-One/cw-simulate-ui
# Fix breaking layout

## Description
Layout would break & wrap right when content grows vertically. Added failsafe to prevent this, instead content will now scroll.

Further fixed growing Stepper, it now becomes scrolling instead.

Also moved screens found in `<Routes>` of `<App>` into dedicated folder

## Test steps
1. Create a new simulation
2. Instantiate contract
3. Run a lot of executions until stepper reaches the end and would start expanding
4. Stepper shouldn't grow and instead get a scrollbar. Layout should remain fixed
